Limelight/ViewControllers/SettingsViewController.m

```diff
@@ -66,6 +67,12 @@ - (void)viewDidLoad {
     [self.bitrateSlider setValue:(_bitrate / BITRATE_INTERVAL) animated:YES];
     [self.bitrateSlider addTarget:self action:@selector(bitrateSliderMoved) forControlEvents:UIControlEventValueChanged];
     [self updateBitrateText];
+    [self.remoteSelector addTarget:self action:@selector(remoteStreamingChanged) forControlEvents:UIControlEventValueChanged];
+}
+
+- (void) remoteStreamingChanged
+{
+    //This function can be used to reconfigure the settings view to offer more remote streaming options (i.e. reduce the audio frequency to 24kHz, enable/disable the HEVC bitrate multiplier, ...)
```

Comment:
Please keep a space between `//` and the start of the comment per the code style of the rest of the project.